CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `com.google.protobuf:protobuf-java` from 3.22.0 to 3.22.2
 - Bump Netty to 4.1.90.Final ([#6677](https://github.com/opensearch-project/OpenSearch/pull/6677)
 - Bump `com.diffplug.spotless` from 6.15.0 to 6.17.0
+- Bump `org.codehaus.jettison:jettison` from 1.5.3 to 1.5.4 ([#6878](https://github.com/opensearch-project/OpenSearch/pull/6878))
```

Comment:
Shouldn't we add this change in unreleased 3.x and then add it back here while backporting to 2.x?

Reply:
The process ideally is, if you know it has to be backported to `x.x`, add it to the `Unreleased x.x` version. That way, the automated backport flow does not fail.

Reply:
@nknize Added the reasoning above. This is also covered within https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#where-should-i-put-my-changelog-entry

> Where should I put my CHANGELOG entry?
Please review the [branching strategy](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#opensearch-branching) document. The changelog on the main branch will contain sections for the next major and next minor releases. Your entry should go into the section it is intended to be released in. In practice, most changes to main will be backported to the next minor release so most entries will likely be in that section.

> The following examples assume the next major release on main is 3.0, then next minor release is 2.5, and the current release is 2.4.

> Add a new feature to release in next minor: Add a changelog entry to [Unreleased 2.x] on main, then backport to 2.x (including the changelog entry).